.github/workflows/ci.yaml

```diff
@@ -61,6 +61,9 @@ jobs:
       - name: Build WASM filter
         run: go run mage.go build
 
+      - name: Build WASM filter without multiphase
```

Comment:
Do you mean to only test with multiphase off?

Reply:
The idea is to build both the filters (with and without multiphase) and then run `mage coverage` that will perform tests against both of them. Mainly I didn't want to delete working tests not compatible anymore with multiphase

Reply:
Ah I should have looked through the magefiles. IIUC, ftw isn't run with multiphase disabled, but it's the most important test and should be run. Also, the filename should be the same regardless of configuration flag.

It'll be better to convert this job to a matrix that runs all the steps with both multiphase off and on to make it clearer all the testing is actually being done, and to not rely on having a different filename.

Reply:
It means that when testing both on and off locally, build/test have to be run twice with/without the env var, but I think that's better than the current approach